src/browser/atom-portable.coffee

```diff
@@ -4,31 +4,34 @@ ipc = require 'ipc'
 
 module.exports =
 class AtomPortable
-  @getPortableAtomHomePath: ->
+  @getPortableAtomHomePath: (platform) ->
+    # Mac has a deeper path for the executable in the application package, so need a different
```

Comment:
*OS X* instead of *Mac*. I realize this is super-pedantic, but we really should start standardizing the platform terminology :bowtie: